lldb/include/lldb/Symbol/Function.h

```diff
@@ -444,8 +444,11 @@ class Function : public UserID, public SymbolContextScope {
 
   Function *CalculateSymbolContextFunction() override;
 
+  // DEPRECATED: Use GetAddressRanges instead.
```

Comment:
Can you make this into a Doxygen comment? It will show up in the generated documentation (for those who use it).

Reply:
Sure.